go/event_test.go

```diff
@@ -24,9 +28,9 @@ func TestEventFrom(t *testing.T) {
 		}
 		token = eventBatch.Token
 		t.Log(fmt.Sprintf("Poll %d out of %d: %d event(s) received", i, maxTries, len(eventBatch.Events)))
-		for _, event := range eventBatch.Events {
-			t.Log(fmt.Sprintf("%s %d %s %s", event.Class, event.ID, event.Operation, event.Ref))
-		}
+		// for _, event := range eventBatch.Events {
```

Comment:
I would suggest removing lines instead of commenting them out. It keeps the code cleaner.